core/src/main/scala/com/databricks/labs/remorph/transpilers/Transpiler.scala

```diff
@@ -3,9 +3,23 @@ package com.databricks.labs.remorph.transpilers
 import com.databricks.labs.remorph.parsers.{intermediate => ir}
 import com.github.vertical_blank.sqlformatter.SqlFormatter
 import com.github.vertical_blank.sqlformatter.core.FormatConfig
+import org.apache.spark.sql.catalyst.parser.CatalystSqlParser
 
 trait Transpiler {
-  def transpile(input: String): String
+  def transpile(input: String, skipValiation: Boolean = false): String
```

Comment:
don't pollute existing interface. validation is a separate stage and has to be a separate interface, if we need it

Reply:
We can align of interface I wanted to come up with small though process on validation strategy for transpiled queries.